src/command-registry.coffee

```diff
@@ -239,6 +250,8 @@ class CommandRegistry
       break if propagationStopped
       currentTarget = currentTarget.parentNode ? window
 
+    @emitter.emit 'did-dispatch', syntheticEvent
```

Comment:
When this gets invoked, `currentTarget` has become `window`, unless `propagationStopped`. See my comment on atom/vim-mode#827.